spec/lib/fetcher_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe Fetcher do
+  let(:server) { 'https://dataspace-dev.princeton.edu/rest/' }
+  let(:collection_id) { '85' } # Astrophysical Sciences
+  let(:fetcher) { described_class.new(server: server) }
+
+  it 'has a server to harvest from' do
+    expect(fetcher).to be_instance_of(described_class)
+    expect(fetcher.server).to eq server
+  end
+
+  context 'fetching data' do
+    let(:oai_feed) { File.read(File.join(fixture_path, 'astrophysical_sciences.json')) }
+
+    before do
+      stub_request(:get, "#{server}/collections/#{collection_id}/items?expand=metadata&limit=100&offset=0")
+        .with(
```

Comment:
Suggestion: I don't think this .with is necessary - these are just default headers the webmock gem outputs, so I think you can remove the .with for a tighter test suite.

Reply:
Good point! Fixed.